TheMovieDb/Views/TagLabel.swift.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  TagLabel.swift.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 31/10/21.
+//
+
+import UIKit.UIImageView
+
+final class TagLabel: UILabel {
```

Comment:
why is this different than a **UILabel**  ? 
seems the only thing you are changing here are properties, other than that is a label, move it instead into a function inside the view you need

Reply:
function created in the view